kotlin/generate-offline-map/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <string name="app_name">Generate offline map</string>
+    <string name="take_map_offline">Take Map Offline</string>
+    <string name="portal_url">https://www.arcgis.com</string>
+    <string name="item_id">acc027394bc84c2fb04d1ed317aac674</string>
+    <string name="offline_map_write_permission_denied">Write permission required to generate offline map</string>
```

Comment:
looks like this resource isn't used, since we're not requiring write permission anymore?

Reply:
good catch!